### DISCUSSION TITLE
Can we use grafana into our react application without IFRAME ??

### DISCUSSION THREAD

**Discussion by a-imran99** [grafana/grafana#39272] [Platform]
Can we use grafana into our react application without IFRAME ??
The plan is to combine the whole grafana including frontend and backend with the react application!

**Comment by torkelo** [comment]
nope, you can use your UI libraries  @grafana/data and @grafana/ui but they are only the base components

**Reply by a-imran99** [reply]
it means you say that we can integrate base UI components and not the entire grafana?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
